public/index.html

```diff
@@ -152,6 +152,258 @@ <h1>Nurture.</h1>
+          software, designed primarily for touchscreen mobile devices such as smartphones and tablets.
+        </p>
+      </div>
+      <!-- Temporary pic -->
+      <img src="./assets/ipad.png" class="product-pic" >
+      <!--  -->
+    </div>
+
+    <div class="product">
+      <div class="product-body">
+        <p class="product-taglines">CREATING IMPACT TOGETHER</p>
+        <p class="product-description">
+          Developer Student Clubs Loyola is a student organization in the Ateneo de Manila University powered 
+          by Google Developers that aims to build students’ skills and network by giving them access to 
+          different technologies, specifically Google Developer technologies like <span>Flutter</span>.
+        </p>
+        <p class="product-notes">
+          A mobile operating system based on a modified version of the Linux kernel and other open source 
+          software, designed primarily for touchscreen mobile devices such as smartphones and tablets.
+        </p>
+      </div>
+      <!-- Temporary pic -->
+      <img src="./assets/ipad.png" class="product-pic" >
+      <!--  -->
+    </div>
+
+    <div class="product">
+      <div class="product-body">
+        <p class="product-taglines">CREATING IMPACT TOGETHER</p>
+        <p class="product-description">
+          Developer Student Clubs Loyola is a student organization in the Ateneo de Manila University powered 
+          by Google Developers that aims to build students’ skills and network by giving them access to 
+          different technologies, specifically Google Developer technologies like <span>Angular</span>.
+        </p>
+        <p class="product-notes">
+          A mobile operating system based on a modified version of the Linux kernel and other open source 
+          software, designed primarily for touchscreen mobile devices such as smartphones and tablets.
+        </p>
+      </div>
+      <!-- Temporary pic -->
+      <img src="./assets/ipad.png" class="product-pic" >
+      <!--  -->
+    </div>
+
+    <div class="product">
+      <div class="product-body">
+        <p class="product-taglines">CREATING IMPACT TOGETHER</p>
+        <p class="product-description">
+          Developer Student Clubs Loyola is a student organization in the Ateneo de Manila University powered 
+          by Google Developers that aims to build students’ skills and network by giving them access to 
+          different technologies, specifically Google Developer technologies like 
+          <span>Google Cloud Platforms</span>.
+        </p>
+        <p class="product-notes">
+          A mobile operating system based on a modified version of the Linux kernel and other open source 
+          software, designed primarily for touchscreen mobile devices such as smartphones and tablets.
+        </p>
+      </div>
+      <!-- Temporary pic -->
+      <img src="./assets/ipad.png" class="product-pic" >
```

Comment:
**add alt text**

Alt text uses:
1. Adding alternative text to photos is first and foremost a principle of web accessibility. Visually impaired users using screen readers will be read an alt attribute to better understand an on-page image.

2. Alt tags will be displayed in place of an image if an image file cannot be loaded.

3. Alt tags provide better image context/descriptions to search engine crawlers, helping them to index an image properly.

Checkout https://moz.com/learn/seo/alt-text#:~:text=Adding%20alternative%20text%20to%20photos,image%20file%20cannot%20be%20loaded.